web_01.html

```diff
@@ -0,0 +1,225 @@
+                    </div>
+                    <div>
+                        <i class="fa-solid fa-mobile-screen-button"></i>
+                        <h1>Responsive</h1>
+                        <p>Developed to keep consistency across platforms and devices.</p>
+                    </div>
+                    <div>
+                        <i class="fa-solid fa-heart"></i>
+                        <h1>Pixel Perfect</h1>
+                        <p>Based on a 12 column grid system, with every pixel in place.</p>
+                        
+                    </div>
+                    <div>
+                        <i class="fa-solid fa-envelope-open-text"></i>
+                        <h1>Fast Support</h1>
+                        <p>Purchasing our templates offers you fast support and regular updates.</p>
+                    </div>
+                </div>
+            </section>
+            <section class="featured__projects">
+                <div class="container">
+                    <div class="project__intro">
+                        <h1>Featured Projects</h1>
+                        <p>Deserunt sit animi totam dicta suscipit non sapiente vero voluptatibus cupiditate.</p>   
+                    </div>
+                    <div class="project__about">
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/typography.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Typography Set</h1>
+                                <p>typography</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/clean_sketch.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Clean Sketch</h1>
+                                <p>creative</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/antic_popcorn.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
```

Comment:
Always keep the parent class name when using BEM
```suggestion
                                <span class="project__left-border"></span>
                                <span class="project__right-border"></span>
```